packages/components/pie-textarea/src/index.ts

```diff
@@ -33,6 +45,21 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
     @validPropertyValues(componentSelector, resizeModes, defaultProps.resize)
     public resize = defaultProps.resize;
 
+    @property({ type: String })
+    public autocomplete?: TextareaProps['autocomplete'];
+
+    @property({ type: Boolean })
+    public autoFocus?: TextareaProps['autoFocus'];
+
+    @property({ type: Boolean })
+    public readonly?: TextareaProps['readonly'];
```

Comment:
I don't think we need `?` on the prop declarations in this file because that should already come from the interface

Reply:
I checked other components and they `?` on the prop declarations
I followed the same code style